c7n/resources/ebs.py

```diff
@@ -128,6 +128,24 @@ def _filter_ami_snapshots(self, snapshots):
     return matches
 
 
+def _filter_kms_active(self, resources):
```

Comment:
Can you move this to a mixin so that this functionality is not duplicated in every resource.